test/test_helper.rb

```diff
@@ -6,13 +6,6 @@
 
 ENV['RAILS_ENV'] = 'test'
 
-module Dummy
-  class Application < Rails::Application
-    config.root = File.join(__dir__, 'dummy')
-    config.load_defaults Rails::VERSION::STRING.to_f
-    config.eager_load = false
-    config.require_master_key = true
-  end
-end
+require_relative 'dummy/config/application.rb'
```

Comment:
D'oh! I copied and pasted the path in my comment, and I forgot to remove the `.rb`.  It's not necessary here.